plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -172,17 +181,21 @@
 {
     private static final Logger log = Logger.get(IcebergMetadata.class);
     private static final String ICEBERG_MATERIALIZED_VIEW_COMMENT = "Presto Materialized View";
+    private static final String ICEBERG_VIEW_COMMENT = "Presto View";
```

Comment:
If the intention is to be compatible with Hive connector (which can be useful for views that reference qualified table names, or during hive/iceberg migration https://github.com/trinodb/trino/pull/8340), we should directly reference constants defined by the hive connector

```
// Be compatible with views defined by the hive connector, which can be useful under certain conditions.
private static final String ICEBERG_VIEW_COMMENT = (a constant extracted from https://github.com/trinodb/trino/blob/8b2dabda9e383527264c5a8d0ed5a16499b67c20/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java#L1851)
```